io/src/ensenso_grabber.cpp

```diff
@@ -50,7 +50,7 @@
 // This function is called whenever an exception is raised to provide details about the error
 void
 ensensoExceptionHandling (const NxLibException &ex,
-                          std::string func_nam)
+                          const std::string &func_nam)
```

Comment:
Do we have any preference to where the & should be placed?  - you vary a bit here 😄

Reply:
I mostly placed them on the left since that is also what our clang-format config says. Here I placed it on the right to match the previous line (io is not formatted yet). But I can also align it left here.

Reply:
Done